lib/content-patcher/patchers/class-imgpatcher.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Patcher for the image elements.
+ *
+ * @package Newspack
+ */
+
+namespace NewspackContentConverter\ContentPatcher\Patchers;
+
+use NewspackContentConverter\ContentPatcher\PatcherInterface;
+use NewspackContentConverter\ContentPatcher\Patchers\PatcherAbstract;
+
+/**
+ * Patcher class for the image elements.
+ *
+ * @package NewspackContentConverter\ContentPatcher\Patchers
+ */
+class ImgPatcher extends PatcherAbstract implements PatcherInterface {
+
+	/**
+	 * See the \NewspackContentConverter\ContentPatcher\Patchers\PatcherAbstract::$patterns for description.
+	 *
+	 * @var array
+	 */
+	protected $patterns = [
+		'match_html_element'     => '/<img.*?>/xim',
+		'match_blocks_element'   => '/<img.*?>/im',
+		'match_attribute_value'  => '/
+			.*?             # anything at the beginning of the string
+			height          # then locate the height attribute
+			\s*             # followed by zero or more spaces
+			=               # the equals char
+			\s*             # once again, with possible zero or more spaces
+			["\\\'+]?       # the attribute value starts with a double or a single quote
```

Comment:
Thank you for spotting this! Yes, both the `+` and the `\` are extra there. They didn't get a chance to do any damage (and probably wouldn't in the future), but they're wrong.

I cleaned them up now. And I also simplified the attribute value matching/capturing part, from `([^"\\']+.*?)` to `([^"']+)`, since only a `+` should do (meaning one or more char except the quote).